internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,46 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
 var (
 	StepLength = 0.65 // длина шага в метрах
 )
 
+var (
+	ErrConvToInt       = errors.New("converting into int error")
+	ErrWrongLenthSlice = errors.New("wrong lenth of slice error")
+	ErrParseDate       = errors.New("parsing date error")
+)
+
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, ErrWrongLenthSlice
+	}
+	// removing spaces
+	parts[0] = strings.TrimSpace(parts[0])
+	parts[1] = strings.TrimSpace(parts[1])
+
+	stepNumber, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, ErrConvToInt
+	}
+
+	t, err := time.ParseDuration(parts[1])
+	if err != nil {
+		return 0, 0, ErrParseDate
```

Comment:
Это конечно, что выделил, но ты теряешь суть ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```